AgentAssistAngular/src/assist-tab/services/assist.service.ts

```diff
@@ -155,45 +157,51 @@ export class AssistService {
-          <div class="action-links">
-              <button class="send-run-btn" id="sendMsg" data-msg-id="${uuids}" data-msg-data="${newTemp}" data-position-id="${positionID}">Send</button>
-              <div class="copy-btn" data-msg-id="${uuids}" data-position-id="${positionID}">
-                  <i class="ast-copy" data-msg-id="${uuids}" data-position-id="${positionID}"></i>
-              </div>
+        <div class="icon_block">
+            <i class="ast-agent"></i>
+        </div>
+        <div class="run-info-content" >
+          <div class="title">Tell Customer</div>
+          <div class="agent-utt">
+            <div class="title-data" id="displayData-${uuids}">${value}</div>
+            <div class="action-links">
+                <button class="send-run-btn" id="sendMsg" data-msg-id="${uuids}" data-msg-data="${newTemp}" data-position-id="${positionID}">Send</button>
+                <div class="copy-btn" data-msg-id="${uuids}" data-position-id="${positionID}">
+                    <i class="ast-copy" data-msg-id="${uuids}" data-position-id="${positionID}"></i>
+                </div>
+            </div>
+            <div class="warning-template hide"  id="warningTemp">
+                <img src="${this.imageFilePath}${this.imageFileNames['infoIcon']}" style="padding-right: 8px;">
+                <span class="warning-text">Templates are not supported, visit your bot builder to disable it.</span>
+            </div>
           </div>
-      </div>
-      </div>
-  </div>
-  `;
-       
+        </div>
+      </div>`;
+
     }else{
       template = `
       <div class="steps-run-data">
-      <div class="icon_block">
-          <i class="ast-agent"></i>
-      </div>
-      <div class="run-info-content" >
-      <div class="title">Tell Customer</div>
-      <div class="agent-utt">
-          <div class="title-data" ><ul class="chat-container" id="displayData-${uuids}" data-msg-id="${uuids}"></ul></div>
-          <div class="action-links">
+        <div class="icon_block">
+            <i class="ast-agent"></i>
+        </div>
+        <div class="run-info-content" >
+          <div class="title">Tell Customer</div>
+          <div class="agent-utt">
+            <div class="title-data" ><ul class="chat-container" id="displayData-${uuids}" data-msg-id="${uuids}"></ul></div>
+            <div class="action-links">
               <button class="send-run-btn" id="sendMsg" data-msg-id="${uuids}" data-msg-data="${newTemp}" data-position-id="${positionID}">Send</button>
               <div class="copy-btn hide" data-msg-id="${uuids}" data-position-id="${positionID}">
                   <i class="ast-copy" data-msg-id="${uuids}" data-position-id="${positionID}"></i>
               </div>
-          </div>
-      </div>
+            </div>
+            <div class="warning-template hide"  id="warningTemp">
+                <img src="${this.imageFilePath}${this.imageFileNames['infoIcon']}" style="padding-right: 8px;">
```

Comment:
@SandeepBa-Kore can't we write padding-right: 8px; in CSS file instead of inline